src/oca_github_bot/tasks/check_cla.py

```diff
@@ -0,0 +1,385 @@
+# Copyright (c) Camptocamp 2018
+# Copyright (c) Akretion 2014
+import json
+import logging
+import re
+import sqlite3
+
+import requests  # FIXME remove
+from requests.auth import HTTPBasicAuth  # FIXME remove
+
+from .. import config
+from ..odoo_client import login
+from ..queue import getLogger, task
+
+_logger = logging.getLogger(__name__)
+RE_WEBLATE_COMMIT_MSG = r"""^Translated using Weblate.*
+Translate-URL: https://translation\.odoo-community\.org/"""
+
+
+_logger = getLogger(__name__)
+
+
+@task()
+def check_cla(event, gh):
```

Comment:
Instead of event, consider passing explicit arguments such as org, repo, pr ([example](https://github.com/OCA/oca-github-bot/blob/e73d881f309a142ded4ee56068433c7b0eb2cc7a/src/oca_github_bot/tasks/tag_approved.py#L18)).

This makes it easier to invoke the task from the command line.

Also, gh can be obtained by reconnecting using global config (the task is potentially running in a different process).